### ISSUE TITLE
Write EFO paper

### ISSUE THREAD

**Issue by paolaroncaglia** [EBISPOT/efo#432]
This is a reminder of our plan for an EFO general paper.
There is some material already that we might link later on.

**Comment by zoependlington** [comment]
Update: Nico, Paola and myself to meet on 25/11/19 to work on paper.

**Comment by matentzn** [comment]
@zoependlington , can you create me a simple tree that makes sense for EFO major categories:

- primary cat 1
  - secondary cat 1
  - secondary cat 2

Something like:

- traits, phenotypes and diseases:
   - disease (EFO:0000408)
   - phenotype (EFO:000)
- anatomy
   - cell type
   - organism
   - anatomical entity

And so on.. Aiming for someting like 4-5 major categories (like in your presentation) and at most 4 sub categories, ideally two or three). Merci bien!

**Comment by paolaroncaglia** [comment]
Update re EFO paper: in progress; Nico worked on it; Paola is looking into details of gathering info from internal and possibly external users; Zoe is addressing a few tasks; Paola and Zoe will generally try to work on draft before the holidays, but unsure how much will get done due to other pressing tasks; will surely reconvene after the holidays to discuss progress and pending matters.

**Comment by matentzn** [comment]
@zoependlington dont forget my comment above! :) I would love to generate the figures tomorrow before I leave, so I can feel you have something to work with to write text.

**Comment by matentzn** [comment]
The data I won't generate (@zoependlington and @paolaroncaglia, if you happen to get additional data from survey of top of your head, fine, if not, just work with the open targets sources):

- [ ] How many sources use EFO overall
- [ ] How many sources use EFO directly (i.e. map directly into EFO)

Less important, only if easy:
- [ ] How many EFO terms are used per source

**Comment by zoependlington** [comment]
Sorry @matentzn, skipped my mind. Here you go!
- Diseases, traits and phenotypes
     - Disease (EFO_0000408)
     - Phenotype (EFO_0000651)
- Anatomy, development and cells
     - organism part (EFO_0000635)
     - cell type (EFO_0000324)
     - developmental stage (EFO_0000399)
- Compounds
     - chemical entity (CHEBI_24431)
- Measurements (could go under traits perhaps?)
     - measurement (EFO_0001444)
- Assays
     - assay (OBI_0000070))

**Comment by paolaroncaglia** [comment]
@zoependlington and @matentzn 
Thanks Zoe for drawing the category tree. EFO also has quite a few protocols, so how about replacing the 'Measurements' category with 'Measurements and protocols' and adding 'protocol' (OBI:0000272) beneath it? That's a relevant area for some of our resources. But maybe not vital, I agree.

**Comment by paolaroncaglia** [comment]
and CL:0000010 'cultured cell' (aka cell lines) under the Anatomy cat please :-)

**Comment by helenp** [comment]
The protocols are likely to be rather old and were originally created for annotation of MAGE-TAB files for gene expression data. I don't know if therefore if this is a current use case for ArrayExpress or any other resource. No objection to the merge.

**Comment by paolaroncaglia** [comment]
Would EFO:0000548 'instrument' be useful? Not sure if those terms are old/currently used.

**Comment by matentzn** [comment]
Zoe and I have worked out a good grouping I think:

![image](https://user-images.githubusercontent.com/7070631/70327481-ca4fa180-182e-11ea-9725-ed5661cac308.png) 

This shows the distribution of entities into five domains: Diseases, Traits and Phenotypes (DTP), Biological processes (BP), Measurement, assays and compounds (MAC), Anatomy, Development and Cells (ADC)  and Other (O)	.

I dont have now time to change it before going on holidays, but lets see whether this works..

**Comment by paolaroncaglia** [comment]
Is there a historical reason why we're using OBI_0100026 for 'organism' instead of NCBITaxon:1 'root', and should we change? Sure, the OBI label is better.

**Comment by helenp** [comment]
Because NCBITaxonomy isn't an ontology, it's a taxonomy and has no commitment to be an ontology. 
Would we do this with a MeSH term?

**Comment by paolaroncaglia** [comment]
@helenp 
No, of course not :-) but all children of OBI_0100026 for 'organism' are NCBITaxon terms, hence my question. Thanks.

**Comment by paolaroncaglia** [comment]
( @matentzn thanks for adding biological process, that would have been my last "request".)

**Comment by zoependlington** [comment]
Sorry! Meant to update my suggested groupings after a chat with Nico earlier to include cultured cell and biological process. These are the new groupings:

- Diseases, traits and phenotypes
     - Disease (EFO_0000408)
     - Phenotype (EFO_0000651)
- Anatomy, development and cells
     - organism part (EFO_0000635)
     - organism (OBI_0100026)
     - cell type (EFO_0000324)
     - developmental stage (EFO_0000399)
     - cultured cell (CL_0000010)
- Measurements, assays and compounds
     - chemical entity (CHEBI_24431)
     - measurement (EFO_0001444)
     - experimental process (EFO_0002694)
- Biological process
     - biological process (GO_0008150)
- Other

**Comment by matentzn** [comment]
Alright @zoependlington @paolaroncaglia I posted some data all across the paper and tagged you for some tasks.. I will stop working now, but feel free to continue tagging me; I will add any more data that you need in the end of the year. I send Zoe a html dump of the current jupyter notebook; I will clean all that up and add it to the supplementary materials when I am back! Good luck for now :)

**Comment by paolaroncaglia** [comment]
Making a note that 
- [ ] "At least four recommended reviewers must be provided by the author at the submission stage" (https://academic.oup.com/bioinformatics/pages/instructions_for_authors).

**Comment by paolaroncaglia** [comment]
@zoependlington
FYI, I'm removing my assignment as I'm sorting out my EFO ticket queue. Of course, I'm available for any discussion on the EFO paper - you'll let me know if there are any tasks that I can address this month. Thank you!

**Comment by jamesamcl** [comment]
**Status triage (2026-08-26):** this ticket has been open since May 2019; a summary of where it stands.

**The ask:** write a general EFO paper (successor to Malone et al. 2010, *Bioinformatics*, PMID:20200009).

**What happened:** there was an active drafting push in Nov–Dec 2019 (category groupings and figures by @zoependlington and @matentzn, journal logistics notes for *Bioinformatics*), a brief follow-up in early 2020, and final housekeeping comments in April–May 2022 pointing material at a "new draft". There has been no activity since 2022-05-11.

**Current situation:** as far as Europe PMC and the web can tell, no new EFO paper (or preprint) has been published — the 2010 Malone et al. paper is still the only citable EFO reference, and the EFO website/docs still direct users to cite it. The closest recent public write-up is the Open Targets "EFO3" blog post, which is not peer-reviewed.

**Recommendation:** the need for an up-to-date citable EFO paper arguably still exists (EFO3, MONDO alignment, and current infrastructure are all post-2010), but the 2019 draft data and figures are now badly outdated and would need regeneration. Suggest the team either (a) recommit with a fresh scope/owner and update the assignee, or (b) close this ticket (or convert it to a discussion/roadmap item) if a paper is no longer planned. Either way, the `in progress issue` label no longer reflects reality.

---
*This is an automated triage review by an AI agent; maintainers, please correct if anything here is wrong.*
